crates/templates/src/renderer.rs

```diff
@@ -99,3 +112,31 @@ fn string_from_bytes(bytes: &[u8]) -> Option<String> {
         Err(_) => None, // TODO: try other encodings!
     }
 }
+
+enum TemplateParseFailure {
+    UnknownFilter(String),
+    Other(liquid::Error),
+}
+
+lazy_static! {
+    static ref UNKNOWN_FILTER: regex::Regex =
+        regex::Regex::new("requested filter=(\\S+)").expect("Invalid unknown filter regex");
```

Comment:
is this `requested filter` string a guaranteed prefix of `liquid::Error` messages?

Reply:
It's what it currently reports when it finds a filter that isn't registered in the parser.  Hopefully no other errors contain that string.  Unfortunately I think this is the best we can do given Liquid's stringy error API.

Reply:
Oh also not a prefix, this is just the substring to look for to pull out the unrecognised name.